File: Examples/DotNET/SDK/VisualBasic/Paragraph/GetParagraphs.vb
Imports Aspose.Cloud

Namespace Paragraph
    Class GetParagraphs
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample.doc"
            Dim subdirection As String = "Paragraph"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, subdirection)

            ' Get a List of Paragraphs from a Word Document
            Dim wordsParagraphsResponse As WordsParagraphsResponse = Common.WordsService.WordsParagraph.GetParagraphs(inputfile, Common.FOLDER)

            Console.WriteLine((Convert.ToString(vbLf) & inputfile) + " have " + wordsParagraphsResponse.Paragraphs.ParagraphLinkList.Count + " paragraphs.")
        End Sub
    End Class
End Namespace

File: Examples/DotNET/SDK/CSharp/Paragraph/GetParagraphs.cs
using System;
using Aspose.Cloud;

namespace CSharp.Paragraph
{
    class GetParagraphs
    {
        public static void Run()
        {
            string inputfile = "doc-sample.doc";
            string subdirection = "Paragraph";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, subdirection);

            // Get a List of Paragraphs from a Word Document
            WordsParagraphsResponse wordsParagraphsResponse = Common.WordsService.WordsParagraph.GetParagraphs(inputfile, Common.FOLDER);

            Console.WriteLine("\n" + inputfile + " have " + wordsParagraphsResponse.Paragraphs.ParagraphLinkList.Count + " paragraphs.");
        }
    }
}
